### DISCUSSION TITLE
error in nextcloud-aio-apache: aborting with incomplete response upstream=localhost:8000

### DISCUSSION THREAD

**Discussion by Jannik44** [nextcloud/all-in-one#3987] [Questions] [answered]
### Steps to reproduce
1. install nextcloud-aio following https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md (nginx-proxy-manager)
2. add the env for domaincheck skipping
3. change storage folder also with the env
4. change apache port to 11000

this is the docker command used:
```
docker run --init --sig-proxy=false --name nextcloud-aio-mastercontainer --restart always --env APACHE_PORT=11000 --env APACHE_IP_BINDING=0.0.0.0 --env NEXTCLOUD_DATADIR="/root/dockerdata/nextcloud" --env SKIP_DOMAIN_VALIDATION=true --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config --volume /var/run/docker.sock:/var/run/docker.sock:ro nextcloud/all-in-one:latest
```

### Expected behavior 
it should work normally
### Actual behavior 
the site runs veryyy slow, to the point where its unusable, it takes more than 30 seconds for a page to load
and i get this error in the nextcloud log like 100times:
 ```ConnectException cURL error 28: Connection timed out after 45002 milliseconds (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://mydomain/hosting/capabilities
Failed to fetch the Collabora capabilities endpoint: cURL error 28: Connection timed out after 45002 milliseconds (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://mydomain/hosting/capabilities 
```
"richdocuments"
and this error in collabora container:
```
wsd-00007-00014 2023-12-24 18:20:16.854602 +0100 [ remotefontconfig_poll ] WRN  #27: Timed out while requesting [GET mydomain/apps/richdocuments/settings/fonts.json] after 30030ms| net/HttpRequest.hpp:1503

wsd-00007-00014 2023-12-24 18:20:16.855080 +0100 [ remotefontconfig_poll ] ERR  Remote config server has response status code: 0 (Unknown)| wsd/COOLWSD.cpp:1306
```
### Host OS 
debian 12
#### Nextcloud AIO version 
v7.9.0
#### Current channel 
latest
#### Other valuable info 
the error message:

```
ERR ts=1703431501.7315042 logger=http.handlers.reverse_proxy msg=aborting with incomplete response upstream=localhost:8000 duration=0.003024417 request={"remote_ip":"172.23.0.3","remote_port":"37766","client_ip":"myip","proto":"HTTP/1.1","method":"GET","host":"mydomain","uri":"/custom_apps/spreed/js/talk-collections.js?v=cb97ce3a-0","headers":{"X-Forwarded-Host":["mydomain"],"User-Agent":["Mozilla/5.0 (X11; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0"],"Sec-Fetch-Mode":["no-cors"],"Cookie":[],"Accept-Encoding":["gzip, deflate, br"],"Sec-Fetch-Site":["same-origin"],"Sec-Fetch-Dest":["script"],"X-Real-Ip":["myip"],"Accept":["*/*"],"Accept-Language":["de,en-US;q=0.7,en;q=0.3"],"X-Forwarded-Scheme":["https"],"X-Forwarded-For":["myip, 172.23.0.3"],"X-Forwarded-Proto":["https"]}} error=writing: write tcp 172.23.0.8:11000->172.23.0.3:37766: write: broken pipe
```

there is also another error in the database container but im not sure if it is related to this problem:
![grafik](https://github.com/nextcloud/all-in-one/assets/76906033/c015661e-9eaf-4a2a-bc44-1516c953d345)

- all nextcloud-aio docker containers are in the same nextcloud docker network and in an additional network where nginx-proxy-manager is
- 
![grafik](https://github.com/nextcloud/all-in-one/assets/76906033/5980dbe6-ebcc-48ef-8f83-be5af09dd7a5)
![grafik](https://github.com/nextcloud/all-in-one/assets/76906033/9444095c-4879-44c5-9374-ab193cdec786)

i should also mention that only most of the request are slow, sometimes the page loads instant on click

and in the redis container it says
```
Memory overcommit is disabled but necessary for safe operation
See https://github.com/nextcloud/all-in-one/discussions/1731 how to enable overcommit
7:C 24 Dec 2023 16:09:28.415 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. Being disabled, it can also cause failures without low memory condition, see https://github.com/jemalloc/jemalloc/issues/1328. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
```
```
	jsresourceloader	
Could not find resource files_external/js/oauth2.js to load 
```

and i should add that im running it on a vps so the nextcloud docker Containers have NO open ports, only the proxy does have open port 80 and 443 wich forwards to the apache container

wsd-00007-00014 2023-12-25 07:08:50.224508 +0100 [ remotefontconfig_poll ] WRN  #27: Timed out while requesting [GET cloud.mchost.fun/apps/richdocuments/settings/fonts.json] after 30009ms| net/HttpRequest.hpp:1503

this message is like 100 times in the collabora container:
```
wsd-00007-00014 2023-12-25 07:08:50.224734 +0100 [ remotefontconfig_poll ] ERR  Remote config server has response status code: 0 (Unknown)| wsd/COOLWSD.cpp:1306
```

edit: i have set up a new instance without collabora and talk, now i dont see any issues, but collabora/talk should get fixed

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?
